app/components/sections/above_fold/navbar/Navbar.tsx

```diff
@@ -30,25 +33,28 @@ const Navbar = ({
       </li>
     ))
   const [toggleMenu, setToggleMenu] = useState<boolean>(false)
+  const [buttonHome, ButtonContact, buttonOther] = buttonText || []
```

Comment:
could "buttonOther" name be more descriptive? i don't understand what buttonOther means?

Reply:
It was resolved